src/domain/useCase/refreshMarketplaceItems.go

```diff
@@ -0,0 +1,14 @@
+package useCase
+
+import (
+	"log/slog"
+
+	"github.com/goinfinite/os/src/domain/repository"
+)
+
+func RefreshMarketplaceItems(marketplaceCmdRepo repository.MarketplaceCmdRepo) {
+	err := marketplaceCmdRepo.RefreshItems()
```

Comment:
I called "Items" on Ez because Marketplace there only has that. One type of item.

Marketplace here has two types of items: installed and catalog. Which one are you referring to?